chipy_org/apps/meetings/models.py

```diff
@@ -96,7 +97,13 @@ def __str__(self):
         MeetingType, blank=True, null=True,
         help_text=("Type of meeting (i.e. SIG Meeting, "
                    "Mentorship Meeting, Startup Row, etc.). "
-                   "Leave this empty for the main meeting."))
+                   "Leave this empty for the main meeting. "
+                   ))
+    custom_title = models.CharField(max_length=64, null=True, blank=True,
+        help_text=("If you fill out this field, this 'custom_title' will show up as the title of the event."
+                   "To have the custom title show up in the Main Meeting on the homepage, select '--' from 'meeting_type' above."
+                   "To have the custom title show up in Upcoming Events on the homepage, select any other 'meeting_type'."  
```

Comment:
I think we can delete these two lines.  Our admins understand that no meeting equals main meeting.  It's hacky; we should open an issue to change that.  I would want Ray to discuss that with the user first.